ngraph/test/backend/minimum.in.cpp

```diff
@@ -79,25 +79,18 @@ NGRAPH_TEST(${BACKEND_NAME}, minimum_int64)
     test_case.run();
 }
 
-// TODO Refactor to use TestCase if u16 will be handled correctly
 NGRAPH_TEST(${BACKEND_NAME}, minimum_u16)
```

Comment:
Can we use template reference tests?
And add new SLT for IE plugins?

Reply:
You also didn't check u8 and u64 data types

Reply:
Done

Reply:
@egorshul  Please remove these backend tests. And use reference tests instead.

Reply:
@ilyachur Reference tests do not work with unsigned type. "Interpreter backend doesn't implement evaluate method for OP Minimum".

Reply:
I think in this case we need to extend evaluate method or add to evaluate_map such data types.

Reply:
@ilyachur Done